index.html

```diff
@@ -0,0 +1,1362 @@
+                </div>
+                <div class="css-zykn71 e55wwux0">
+                    <div>
+                        <span class="css-1s40g2l">Dow</span></div><div class="css-x1sij0"><div direction="negative" class="css-4dbexh e55wwux1">-0.88%</div><svg width="5px" height="9px" viewBox="0 0 5 9" class="css-4ky835"><title>Group</title><defs></defs><g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g id="Artboard"><g id="Group"><path d="M2.5,9 L2.5,3.373" id="Shape" stroke="#a61b1e"></path><polyline id="Shape" fill="#a61b1e" fill-rule="nonzero" points="4.86293961 4.09179688 2.5 0 0.13706039 4.09179688"></polyline></g></g></g></svg>
+                    </div>
+                </div>
+           
+            </div>
+
+
+        </div>
+
+         <!-- End of logo nav -->
+
+        <hr class="hr_one">
+
+        <div class="timeline">
+            <div class="tl_date">
+                <h4>Friday, June 3, 2022</h4>
+            </div>
+            <div class="tl_sub">
+                <p>SUBSCRIBE FOR ₹600/YEAR</p>
+            </div>
+        </div>
+
+        
+
+
+
+        <div class="nav_list">
+            <ul class="cont">
```

Comment:
follow the scss class naming convention. Ul should be nav_list_container instead of cont.